README.md

```diff
@@ -1,2 +1,8 @@
-# example-azure-pipelines
-An example repo demonstrating how to integrate srcclr agent with Azure Pipelines
+# [:] Using SourceClear via Azure DevOps
```

Comment:
The formal name is Azure Pipelines instead of Azure DevOps

Reply:
changes should be reflected in commit 4e64bb3